src/shared/dumb_components/field.js

```diff
@@ -11,23 +11,23 @@ var Field = (props) =>
   ? <Input {...props} />
   : <div></div>
 
-var Display = ({ name, id, value, description }) =>
+var Display = ({ name, id, value, description, memberView }) =>
   <p>
     {description && name === 'News type: ' && <span><br /><i>{description}</i><br /></span>}
-    <br />
+    {memberView && <br />}
```

Comment:
We should drop the coupling in the `id`, allow a class to be passed in and the difference in spacing should be done in the CSS